.github/workflows/README.md

```diff
@@ -89,25 +79,25 @@ TODO
 - ##### `backend-unit-test`:
 
   - Checks if containers need to be rebuilt based on diffs on docker + requirements files
-  - Runs tests in docker-compose: `make local-init-test-data & make all-local-unit-test-backend`
-    - `make local-init-test-data` -> `docker-compose $(COMPOSE_OPTS) run --rm -T backend /bin/bash -c "pip3 install awscli && cd /single-cell-data-portal && scripts/setup_dev_data.sh"`
-    - `make all-local-unit-test-backend` -> (inside backend container) `make container-unittest;`
-      - `make container-unittest` -> `DEPLOYMENT_STAGE=test PYTHONWARNINGS=ignore:ResourceWarning python3 \ -m unittest discover --start-directory tests/unit/backend --top-level-directory . --verbose;`
+  - Runs tests in docker-compose: https://github.com/chanzuckerberg/single-cell-data-portal/blob/6a423183c255737d2a44e40447a91d0ece041a41/.github/workflows/push-tests.yml#L214-L216
+    - https://github.com/chanzuckerberg/single-cell-data-portal/blob/6a423183c255737d2a44e40447a91d0ece041a41/Makefile#L106-L107
+    - https://github.com/chanzuckerberg/single-cell-data-portal/blob/6a423183c255737d2a44e40447a91d0ece041a41/Makefile#L182-L185
+      - https://github.com/chanzuckerberg/single-cell-data-portal/blob/6a423183c255737d2a44e40447a91d0ece041a41/Makefile#L27-L30
 
 - ##### `processing-unit-test`:
```

Comment:
I think one-liner descriptions for all the test types like 'Sets up docker container to run through a dataset submission job and validates behavior" would be helpful